install.md

```diff
@@ -197,27 +194,41 @@ To configure the installation settings:
 
       * Create a secret.
 
-      ```bash
-      kubectl create namespace APP-TLS-SECRET-NAMESPACE
-      kubectl delete secret APP-TLS-SECRET-NAME --namespace APP-TLS-SECRET-NAMESPACE
-      kubectl create secret tls APP-TLS-SECRET-NAME \
-        --cert=tls.crt \
-        --key=tls.key \
-        --namespace APP-TLS-SECRET-NAMESPACE
-      ```
+        ```bash
+        kubectl create namespace APP-TLS-SECRET-NAMESPACE
+        kubectl create secret tls APP-TLS-SECRET-NAME \
+          --cert=tls.crt \
+          --key=tls.key \
+          --namespace APP-TLS-SECRET-NAMESPACE
+        ```
 
-1. If you do not already have a secret containing the hostname, username, and password for your application image registry, create one:
+1. If you do not already have a secret containing the hostname, username, and password for your application image registry:
+   * Create a secret.
 
       ```bash
       kubectl create namespace APP-REGISTRY-CREDENTIALS-SECRET-NAMESPACE
-      kubectl delete secret APP-REGISTRY-CREDENTIALS-SECRET-NAME --namespace APP-REGISTRY-CREDENTIALS-SECRET-NAMESPACE
       kubectl create secret docker-registry APP-REGISTRY-CREDENTIALS-SECRET-NAME \
         --docker-server=APP-REGISTRY-HOSTNAME \
         --docker-username=APP-REGISTRY-USERNAME \
         --docker-password=$(cat /path/to/APP-REGISTRY-PASSWORD) \
         --namespace APP-REGISTRY-CREDENTIALS-SECRET-NAMESPACE
       ```
 
+   * Create a SecretExport to allow TAS Adapter to copy the secret into the "cf" namespace.
```

Comment:
Should this be a separate top-level item, independent of the existence of the registry-credential secret itself? A platform operator might have that secret somewhere, but would still need to supply the SecretExport resource to match the installation's SecretImport.

Reply:
@emalm I pulled it up. Look good?

Reply:
Yes, thank you!